### PR TITLE
Multilevel ibfe 2

### PR DESCRIPTION
**Pull request by drwells**
This PR updates our data structures in FEDataManager so that we can also index by patch level.

### IBAMR Pull Request Checklist
- [x] Does the test suite pass?
- [x] Was clang-format run?
- [x] Were relevant issues cited? Please remember to add `Fixes #12345` to close
      the issue automatically if we are fixing the problem.
- [x] Is this a change others will want to know about? If so, then has a
      changelog entry been added?
- [x] If new data structures have been added to a class then have they been set
      up appropriately for restarts? If so, ensure that the restart version
      number is incremented.
- [x] Does this change include a bug fix or new functionality? If so, a new test
      or tests should be added. New tests should run quickly (less than a minute
      in release mode). If possible, an older test should gain a new option so
      that we do not need to compile more test executables.
- [x] Did you (if your account has permission to do so) set relevant labels on
      GitHub for the pull request?

### CODE REVIEW

**Comment by drwells** [comment]
Most changes are due to whitespace - look at https://github.com/IBAMR/IBAMR/pull/1108?w=1 to see the interesting parts.

**Comment by boyceg** [comment]
@drwells did you say there is a way to get GitHub to make it easier to review whitespace-dominated changes?

**Comment by boyceg** [comment]
(Nevermind --- I figured it out!)

File: ibtk/src/lagrangian/FEDataManager.cpp [hunk 1]
```diff
@@ -496,13 +496,13 @@ FEDataManager::getDefaultSpreadSpec() const
 const std::vector<std::vector<Elem*> >&
 FEDataManager::getActivePatchElementMap() const
 {
-    return d_active_patch_elem_map;
+    return d_active_patch_elem_map.back();
```
Comment (drwells) [inline ibtk/src/lagrangian/FEDataManager.cpp:499]: While not ideal this is necessary to keep IBFEMethod and IBFESurfaceMethod happy at the moment.

**Review by boyceg — APPROVED** [review]
Looks good!

Seems like this is almost there.

File: ibtk/src/lagrangian/FEDataManager.cpp (hunk 1 above)
Comment (boyceg) [reply ibtk/src/lagrangian/FEDataManager.cpp:499]: I was going to comment on that. :-D

Comment (drwells) [reply ibtk/src/lagrangian/FEDataManager.cpp:499]: I don't want to break the IIM branch even more badly than I already have

**Pull request merged**